core/vm/operations_verkle.go

```diff
@@ -132,6 +132,10 @@ func gasExtCodeCopyEIP4762(evm *EVM, contract *Contract, stack *Stack, mem *Memo
 		return 0, err
 	}
 	addr := common.Address(stack.peek().Bytes20())
+
+	if _, isPrecompile := evm.precompile(addr); isPrecompile {
```

Comment:
Does L130 makes sense for precompiles? 
I see that [gasExtCodeCopyEIP2929](https://github.com/gballet/go-ethereum/blob/fbbb499dda8c70cae13e238d9209722465e0a033/core/vm/operations_acl.go#L125) doesn't make a distinction either, so I guess is coherent. Just wondering.

Reply:
yeah, because the way `CODECOPY` works, if no code can be found, it will right-pad the data and fill RAM.